app/assets/config/sidekiq_monitoring_manifest.js

```diff
@@ -0,0 +1,2 @@
+//= link_directory ../javascripts/sidekiq_monitoring .js
+//= link_directory ../stylesheets/sidekiq_monitoring .css
```

Comment:
why do we need css/js ? couldn't this all go?

Reply:
Yess, I'll throw it all out! 😄